test/unit/org/apache/cassandra/db/marshal/AbstractTypeTest.java

```diff
@@ -460,6 +460,15 @@ public void nested()
         });
     }
 
+    @Test
+    @SuppressWarnings("rawtypes")
+    public void nestedDuration()
+    {
+        qt().withShrinkCycles(0).forAll(AbstractTypeGenerators.builder().withoutTypeKinds(PRIMITIVE, COUNTER).withPrimitives(DurationType.instance).build()).checkAssert(type -> {
```

Comment:
```suggestion
        qt().forAll(AbstractTypeGenerators.builder().withoutTypeKinds(PRIMITIVE, COUNTER).withPrimitives(DurationType.instance).build()).checkAssert(type -> {
```

we disable shrinks by default so you don't need to in the test

Reply:
this generator doesn't produce `ReversedType`, only the table generator does that...